frontend/src/components/ContentDisplayToggle.tsx

```diff
@@ -0,0 +1,29 @@
+import { Dispatch, SetStateAction } from "react";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+// this could implemented slightly differently to either be simpler and more generic
+// or we could opt to build the entire show / hide functionality into this component and pass the content controlled by the toggle as children
+export default function ContentDisplayToggle({
+  setToggledContentVisible,
+  toggledContentVisible,
+  toggleText,
+}: {
+  setToggledContentVisible: Dispatch<SetStateAction<boolean>>;
+  toggledContentVisible: boolean;
+  toggleText: string;
+}) {
+  const iconName = toggledContentVisible ? "arrow_drop_up" : "arrow_drop_down";
+  return (
+    <div data-testid="content-display-toggle" className="grid-col-4">
+      <button
```

Comment:
+1 on using an unstyled uswds button instead of an anchor tag, I know we discussed that yesterday